resources/views/contents/_gallery.blade.php

```diff
@@ -1,19 +1,15 @@
 @if(!empty($images) && $images->isNotEmpty())
     <div class="row image-gallery">
         @foreach($images as $image)
-            <?php $imageTranslation = $image->translation($lang->code); ?>
-            <?php $thumbId = isset($thumb) ? $thumb->id : null;    ?>
-            @if($image->id !== $thumbId)
-                <div class="col-xs-6 col-md-3 image">
-                    <a href="{{croppaUrl($image->getFullPath())}}" class="colorbox thumbnail"
-                       rel="gallery" title="{{($imageTranslation)? $imageTranslation->title : ''}}">
-                        <img class="img-fluid"
-                             title="{{($imageTranslation)? $imageTranslation->title : ''}}"
-                             src="{{croppaUrl($image->getFullPath(), 180, 180)}}"
-                             alt="{{($imageTranslation)? $imageTranslation->title : ''}}">
-                    </a>
-                </div>
-            @endif
+            <div class="col-xs-6 col-md-3 image mb-4">
+                <a href="{{croppaUrl($image->uploadPath())}}" class="colorbox thumbnail"
+                   rel="gallery" title="{{($image->title($language->code))? $image->title($language->code) : ''}}">
+                    <img class="img-fluid"
+                         title="{{($image->title($language->code))? $image->title($language->code) : ''}}"
```

Comment:
You don't need to check if title exists, view model does it for you and it will return null if title is not available

Reply:
done